<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MatchmakingPlayerInfo</class>
 <widget class="QWidget" name="MatchmakingPlayerInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>217</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignTop">
    <widget class="QLabel" name="lblLocalPlayerLeagueImage">
     <property name="maximumSize">
      <size>
       <width>96</width>
       <height>96</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/img/client/rank_icons/league_1</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" alignment="Qt::AlignTop">
    <widget class="QFrame" name="frmDivision">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignTop">
       <widget class="QLabel" name="lblDivision">
        <property name="font">
         <font>
          <family>Noto Sans</family>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Division Name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" alignment="Qt::AlignHCenter|Qt::AlignTop">
       <widget class="QLabel" name="lblPoints">
        <property name="font">
         <font>
          <family>Noto Sans</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>0,000 Points</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" alignment="Qt::AlignHCenter|Qt::AlignTop">
    <widget class="QLabel" name="lblPlayerName">
     <property name="styleSheet">
      <string notr="true">font-family: &quot;Gobold Bold&quot;;
font-size: 24px;</string>
     </property>
     <property name="text">
      <string>Player Name</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
